### ISSUE TITLE
[BUG] Prod build doesn't work with Angular 6.1.6

### ISSUE THREAD

**Issue by Dzhuneyt** [cornflourblue/jw-angular-social-buttons#3]
```
ERROR in : Unexpected value 'JwSocialButtonsModule in C:/htdocs/project/node_modules
/jw-angular-social-buttons/lib/jw-angular-social-buttons.module.d.ts' imported by the m
odule 'AppModule in C:/htdocs/project/src/app/app.module.ts'. Please add a @NgModule
 annotation.
```

This is the error that I get after following the examples in the article: http://jasonwatmore.com/post/2018/06/01/angular-2-social-sharing-buttons-for-facebook-google-plus-twitter-linkedin-and-pinterest

**Comment by KewalMehta** [comment]
Any update on above issue ? @Dzhuneyt did  u resolved it ? I am also facing the same issue.

**Comment by cornflourblue** [comment]
This should fixed now with version 1.0.0 that I just published to npm. The module was missing `*.metadata.json` files which are required for AOT / prod compilation to work correctly

**Comment by mohitkr5** [comment]
It does not work in Angular 8, is there any resolution?
